[tests] Fix flake8 warnings in several wallet functional tests

jnewbery opened the pull request:
Fixes flake8 warnings in several wallet functional tests.

Several wallet functional tests need rewrite to remove the accounts API (#13075). To prepare for that, I fixed all the flake8 warnings in those tests.

#13075 is blocked on a bitcoind bug. This PR is just the flake8 fixes so we're not completely blocked.

jnewbery committed:
[tests] Fix flake8 warnings in several wallet functional tests

This commit fixes flake8 warnings in the following functional tests:

- wallet_listreceivedby.py
- wallet_basic.py
- wallet_txn_clone.py
- wallet_listsinceblock.py
- wallet_import_rescan.py
- wallet_listtransactions.py
- wallet_importprunedfunds.py
- wallet_txn_doublspend.py

MarcoFalke commented on test/functional/wallet_basic.py:
```diff
@@ -215,89 +227,89 @@ def run_test(self):
+        # 4. check if recipient (node0) can list the zero value tx
         usp = self.nodes[1].listunspent()
-        inputs = [{"txid":usp[0]['txid'], "vout":usp[0]['vout']}]
+        inputs = [{"txid": usp[0]['txid'], "vout":usp[0]['vout']}]
```
If you add a space here, then make it consistently for the second kv as well

MarcoFalke commented on test/functional/wallet_listtransactions.py:
```diff
@@ -129,21 +136,21 @@ def get_unconfirmed_utxo_entry(node, txid_to_match):
 
         # Create tx2 using createrawtransaction
-        inputs = [{"txid":utxo_to_use["txid"], "vout":utxo_to_use["vout"]}]
+        inputs = [{"txid": utxo_to_use["txid"], "vout":utxo_to_use["vout"]}]
```
same here

MarcoFalke commented:
utACK 5ec11f44f322be851746903578bd5224b9807e51. I am not aware of any significant pull request that would conflict due to this, so it seems fine.

ryanofsky reviewed (commented):
utACK 5ec11f44f322be851746903578bd5224b9807e51

jnewbery commented:
@MarcoFalke - addressed your comments. I've also fixed a couple of variable names which I somehow missed before.

@ryanofsky - Thanks for the quick review. `git diff 5ec11f4 a533834d` should be minimal

MarcoFalke commented:
re-utACK a533834d502ef72aa14216828ae0ff5c7fc5625e

ryanofsky reviewed (commented):
utACK a533834d502ef72aa14216828ae0ff5c7fc5625e

MarcoFalke commented:
Note that this is no longer [trivial] since you change variable names and remove assignments...
